slsim/lsst_science_pipeline.py

```diff
@@ -745,6 +745,106 @@ def dp0_time_series_images_data(butler, center_coord, radius="0.1", band="i", si
     return table_data
 
 
+def opsim_time_series_images_data(
+    butler, center_coord, radius="0.1", band="i", size=101
+):
+    """Creates time series data from opsim database.
+
+    :param butler: butler object
+    :param center_coord: A coordinate point around which we need to create time series
+        images.
+    :param radius: radius for query
+    :param band: imaging band
+    :param size: cutout size of images
+    :return: An astropy table containg time series images and other information
+    """
+
+    expo_information = tap_query(center_coords=center_coord, radius=radius, band=band)
+    calexp_image = list_of_calexp(expo_information, butler=butler)
+    radec = dp0_center_radec(calexp_image[0])
+    # Question: is radec_list only for 1 observation? Or can it be a list with multiple ra and decs?
+    # It might be better here to skip the butler and just get ra, dec from center_coord
+    radec_list = [(radec.getRa().asDegrees(), radec.getDec().asDegrees())]
+
+    # Everything in this section below might be better in a separate opsim function
+    # ------------------------------------------------------------------
+    # Users need to have the right branch of opsimsummary installed (Issue#325/proposalTables)
+    # and the opsim database downloaded
+    from opsimsummary import SynOpSim
```

Comment:
here you can perhaps write it as:
try:
from opsimsummary ....
except:
raise ImportError("Users need to have the right branch of opsimsummary installed (Issue#325/proposalTables)
    # and the opsim database downloaded")
or something of this form